### PR TITLE
Added SCP:SL Support

### PR DESCRIPTION
**Pull request by Redolant**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [x] Does your server use a custom docker image? 
    * [x] Have you tried to use a generic image?
    * [x] Did you PR the nesessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: scpsl/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# SCPSL-Pterodactyl-Egg
+
+A Pterodactyl Egg for SCP: Secret Laboratory
+
+This game needs about a minimal of 3096MB of RAM to run smooth.
+
+# How to import this egg?
```
Comment (parkervcp) [inline scpsl/README.md]: This and below isn't really necessary as it's in the main repo and we call it out all over the discord.

Comment (Vyrezz) [reply scpsl/README.md]: I changed the readme

**Comment by TrollikCz** [comment]
Hello i tested this and i think there is some problems with it because something is with encoding because i have "?" in console in place where is letter like this "ň"

**Comment by parkervcp** [comment]
@Redolant Any updates?

**Comment by gOOvER** [comment]
I think, this can be closed, until we have 2 Eggs for it

**Pull request closed without merging**